src/stylesheets/app.css

```diff
@@ -148,14 +147,30 @@ button.loader:hover {
 
 .meta-container {
   text-align: right;
-  width: 80%;
+  width: 90%;
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  align-items: baseline;
+  column-gap: 7px;
+}
+
+.meta-container a {
+  text-align: left;
+}
+
+.deploy-view {
+  display: grid;
 }
 
 .publish-button-container {
-  width: 50%;
-  margin: 0 auto;
+  justify-self: center;
   text-align: center;
 }
+
+#publish-button {
+  width: 150px;
```

Comment:
What do we think of setting a `min-width` instead here? This way, in case the window/app ever re-sizes, we keep some flexibility

Reply:
Ah I see, I'll do a quick fix for that